src/tests/api-tests.ts

```diff
@@ -5589,7 +5592,7 @@ describe('api tests', () => {
     );
     expect(unanchoredResult.status).toBe(200);
     expect(unanchoredResult.type).toBe('application/json');
-    expect(JSON.parse(unanchoredResult.text).total).toEqual(60); // 60 txs up to unanchored block_height=3
+    expect(JSON.parse(unanchoredResult.text).total).toEqual(50); // 60 txs up to unanchored block_height=3
```

Comment:
60 should be the correct value here. I'm not able to check right now but I believe removing the LIMIT/OFFSET from `WITH stx_txs AS` would make this expectation valid again (left comment there).

Reply:
The above http query for this specifies `?limit=50`. That limit query was fixed and required this be fixed (or we could change the limit query param but that errors when set to above 50 atm).

Reply:
IIUC, 50 should be the page size (thus the JSON `results` array size, specified by `limit=50`) but 60 is the total number of txs for that principal (the `total` JSON key, gathered by the `countOverColumn`), right?

Reply:
Yeah, you're right. This change was fixed/reverted